modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -56,6 +58,14 @@ def __init__(self, object_id, length=None, width=None, ip=None, call_queue=None)
         self._length_cache = length
         self._width_cache = width
         self._ip_cache = ip
+        self._identity = uuid.uuid4().hex[8:]
+
+        logger = get_logger()
+        logger.debug(
+            "Partition ID: {}, Height: {}, Width: {}, Node IP: {}".format(
```

Comment:
Should we do a check if these are None, and print something more useful than `Width: None` e.g. if they are (maybe like width has not been initialized or something)?

Reply:
For now, the logs are a fairly advanced feature and I think printing `None` is not totally necessary for the partition metadata. We will have more detailed Ray remote logging for the partitions in the future anyhow that @jeffreykennethli is working on.